tests/test_imports.py

```diff
@@ -5,13 +5,14 @@
 
 import os
 import sys
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'misc'))
+TESTS_FAKE_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), 'misc', 'test.py'))
```

Comment:
The reason why I'm not changing the `sys.path` anymore is because I had some problems with it while running tests. It caused some tests to not work or to work in non deterministic ways...
So I decided to use memestra's API to pass paths.

Reply:
+1